### ISSUE TITLE
NestableScrollContainer with NestableDraggableFlatList, VirtualizedLists should never be nested inside plain ScrollViews with the same orientation

### ISSUE THREAD

**Issue by lunajing** [computerjazz/react-native-draggable-flatlist#393]
I use NestableScrollContainer and NestableDraggableFlatList exactly as in the example, but I get the following scrollview nesting error in console：

`VirtualizedLists should never be nested inside plain ScrollViews with the same orientation because it can break windowing and other functionality - use another VirtualizedList-backed container instead.`

my code: 
`<NestableScrollContainer>
      <NestableDraggableFlatList
        scrollEnabled={false}
        data={data1}
        renderItem={renderItem}
        keyExtractor={(index) => `${index}`}
        onDragEnd={({ data }) => setData1(data)}
      />
      <NestableDraggableFlatList
        scrollEnabled={false}
        data={data2}
        renderItem={renderItem}
        keyExtractor={(index) => `${index}`}
        onDragEnd={({ data }) => setData2(data)}
      />
      <NestableDraggableFlatList
        scrollEnabled={false}
        data={data3}
        renderItem={renderItem}
        keyExtractor={(index) => `${index}`}
        onDragEnd={({ data }) => setData3(data)}
      />
    </NestableScrollContainer>`

**Platform & Dependencies**
 - react-native-draggable-flatlist version: 3.1.2
 - Platform: iOS/Android
 - React Native: 0.69
 - Reanimated version: 2.8.0
 - React Native Gesture Handler version: 2.2.1

**Comment by web3Liuwentong** [comment]
+1

**Comment by isuru102** [comment]
I am getting the same error

**Comment by lunajing** [comment]
Has anyone paid attention to this issue?

**Comment by computerjazz** [comment]
yeah, you're gonna just have to ignore if you want to nest flatlists within scrollviews. The issue is that if you want flatlists-within-scrollables, virtualization will no longer work, since the list will expand to fill all available space.

**Comment by JaRoMasterDev** [comment]
Is there a way to ignore that warning with LogBox?

**Comment by CaCaBlocker** [comment]
+1

**Comment by goughjo02** [comment]
try nesting the NestableScrollContainer in a view as in their example https://github.com/computerjazz/react-native-draggable-flatlist/blob/main/Example/screens/NestedScreen.tsx . it got rid of the error for me

**Comment by EstivenGonzalez** [comment]
+1
